src/utils/strings.js

```diff
@@ -10,4 +10,41 @@ function snakeCaseToStartCase(str) {
     .join(' ');
 }
 
-export { snakeCaseToStartCase };
+/**
+ * Formats multiple session numbers into a string
+ * @param {Array<string>} sessionNumbers - Array of session numbers
+ * @returns {string} Formatted string of session numbers
+ */
+function formatMultipleSessionNumbers(sessionNumbers) {
```

Comment:
I don't think this is limited to session numbers, so I think we can remove mention of sessions.

Reply:
genericalized it